doc/api/n-api.md

```diff
@@ -957,7 +956,7 @@ To define a class so that new instances can be created (often used with
 napi_value Init(napi_env env, napi_value exports) {
   napi_status status;
   napi_property_descriptor properties[] = {
-    { "value", NULL, GetValue, SetValue, 0, napi_default, 0 },
+    { "value", 0, 0, GetValue, SetValue, 0, napi_default, 0 },
```

Comment:
I thought the suggestion was to use NULL instead of 0, just wondering why you went the other way?

Reply:
Oh, right. I thought it would be more consistent, but you're right. This is plain C. I should actually use `NULL` everywhere – though that might expand the scope of the PR.

I'll add `NULL`s everywhere this PR touches, and do another sweep to make sure that the entire doc says `NULL`.